.github/workflows/peek_icons.yml

```diff
@@ -9,12 +9,12 @@ jobs:
     # fail due to system, skipped
     name: Peek Icons
     if: github.event.label.name == 'bot:peek'
-    runs-on: windows-2019
+    runs-on: windows-latest
```

Comment:
Wait, why does this run on windows?

Reply:
It was due to the Geckodriver being used in Windows only.
We can change it to Linux, but I don't see any difference at all.

Reply:
Linux is faster, more stable, more frequently updated and more storage efficient.
So if we can use it, we should :)

Reply:
Changed to use `ubuntu-latest` as well as fixed an issue with the workflow.